samples/angular-todo/src/app/features/journey/form/form.component.ts

```diff
@@ -161,7 +199,17 @@ export class FormComponent implements OnInit {
    */
   handleStep(step?: FRStep) {
     this.step = step;
-
+    const selectIdCallback = step.callbacks.find((cb) => cb.payload.type === 'SelectIdPCallback');
+    const callBackInputValue = selectIdCallback?.payload.input[0].value as string;
+    if (selectIdCallback && callBackInputValue === '') {
+      selectIdCallback.payload.input[0].value = 'localAuthentication';
+    }
+    const redirectCallback = step.callbacks.find(
+      (callback) => callback.getType() === CallbackType.RedirectCallback,
+    );
+    if (redirectCallback) {
+      FRAuth.redirect(step);
+    }
```

Comment:
This code is a bit dense. Can we get some code comments in here expressing intent? It's worth remembering the purpose of these sample apps, and that is to educate our customers on how to implement features. To do that, we need to a liberal amount of comments to guide the reader along.

Reply:
done

Reply:
i had to add this workaround because it crashes if it was a local authentication, but now is no longer needed. It is taken care of at line 100. removed it, made a few tests and works